### ISSUE TITLE
Huddles branch incomplete

### ISSUE THREAD

**Issue by noranda** [intervention-engine/frontend-react#28]
### `huddles` branch incomplete

WORKING:
* [x] `huddle` action and reducer -- not tested, using dummy data
* [x] Care Team and Huddle filter on patient list including clickable dates
* [x] Next huddle date on patient list
* [x] Next huddle date on patient view
* [x] Care Team selection on patient view
* [x] Huddle calendar on patient view including styles for scheduled huddles

NOT WORKING:
* [ ] Huddle details for selected date on calendar
* [ ] Adding a patient to a huddle -- function written but not tested, API not yet done
* [ ] Removing a patient from a huddle

All tests passing. Once branch is complete, merge into `development`.
